vscode-inferno-syntax-highlighting/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 PanAeon
```

Comment:
???

Reply:
Copied from the existing license file in `vscode-inferno-lsp-server` (there was no license before). I also found the attribution strange, I'll change both of them